src/js/components/bulkDownload/BulkDownloadPage.jsx

```diff
@@ -20,15 +21,21 @@ import BulkDownloadModalContainer from
 import BulkDownloadSidebar from './sidebar/BulkDownloadSidebar';
 
 export const dataTypes = [
+    {
```

Comment:
Can we get these from `dataMapping/navigation/menuOptions.js` to ensure that labeling and enable/disable state is centrally managed?

Reply:
Yep